docs-source/content/samples/simple/soa-preview-guide/_index.md

```diff
@@ -0,0 +1,1216 @@
+
+
+##### Create the database
+
+To create the database pod and service, you need to create a Kubernetes YAML
+file similar to the one shown below.  This example is provided in the WebLogic
+Kubernetes operator repository in this location:
+
+`kubernetes/samples/scripts/create-soa-domain/create-database/db-with-pv.yaml`
+
+{{% notice warning %}}
+**TODO FOR MARK**  
+In `develop` branch, this file is moved to:
+`kubernetes/samples/scripts/create-soa-domain/domain-home-on-pv/create-database/db-with-pv.yaml`  
+Need to update this after that change is merged to `master`
+{{% /notice %}}
+
+Here are the contents of the example file:
+
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  name: soadb
+  labels:
+    app: soadb
+  namespace: soans
+spec:
+  ports:
+  - port: 1521
+    name: server-port
+  - port: 5500
+    name: em-port
+  clusterIP: None
+  selector:
+    app: soadb
+---
+apiVersion: apps/v1beta1
+kind: StatefulSet
+metadata:
+  name: soadb
+  namespace: soans
+spec:
+  serviceName: "soadb"
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: soadb
+    spec:
+      terminationGracePeriodSeconds: 30
+      containers:
+      - name: soadb
+        image: coantiner-registry.oracle.com/database/enterprise:12.2.0.1
+        imagePullPolicy: IfNotPresent
+        ports:
+        - containerPort: 1521
+          name: server-port
+        - containerPort: 5500
+          name: em-port
+        env:
+        - name: DB_SID
+          value: soadb
+        - name: DB_PDB
+          value: soapdb
+        - name: DB_DOMAIN
+          value: my.domain.com
+        - name: DB_BUNDLE
+          value: basic
+        readinessProbe:
+          exec:
+            command:
+            - grep
+            - "Done ! The database is ready for use ."
+            - "/home/oracle/setup/log/setupDB.log"
+          initialDelaySeconds: 300
+          periodSeconds: 5
+        volumeMounts:
+        - mountPath: /ORCL
+          name: soadb-storage
+      volumes:
+      - name: soadb-storage
+        persistentVolumeClaim:
+          claimName: soadb-pvc
+```
+
+If you are using the Oracle Container Engine for Kubernetes and you created the persistent
+volume claim as shown above, you will need to add an `initContainer` to change the
+owner of the file system to the correct `oracle` user that the Oracle Database image
+uses (which is uid 54321).  You can do this by adding the lines shown below:
+
+```yaml
+    ... lines omitted ...
+    spec:
+      terminationGracePeriodSeconds: 30
+      # add the lines after this line
+      initContainers:
+      - name: fix-pvc-owner
+        image:  busybox
+        command: ["sh", "-c", "chown -R 54321:54321 /ORCL"]
+        volumeMounts:
+        - name: soadb-storage
+          mountPath: /ORCL
+      # add the lines before this line
+      containers:
+      - name: soadb
+        image: coantiner-registry.oracle.com/database/enterprise:12.2.0.1
```

Comment:
type in image string

Reply:
typo in my comment too :-)

Reply:
Got it thanks (there were two of those).